README.md

```diff
@@ -65,7 +155,9 @@ docs/formatter.rb linguist-documentation=false
 
 #### Generated code
 
-Not all plain text files are true source files. Generated files like minified js and compiled CoffeeScript can be detected and excluded from language stats. As an added bonus, unlike vendored and documentation files, these files are suppressed in diffs.
+Not all plain text files are true source files. Generated files like minified JavaScript and compiled CoffeeScript can be detected and excluded from language stats. As an added bonus, unlike vendored and documentation files, these files are suppressed in diffs. [`generated.rb`](/lib/linguist/generated.rb) lists common generated paths and excludes them from the language statistics of your repository.
+
+Use the `linguist-generated` attribute to mark or unmark paths as generated.
```

Comment:
We could probably link `linguist-generated` to the relevant section of `README.md` which addresses overrides.

Reply:
This _is_ in that section 😄. I don't think we need to link to itself.

Reply:
Oh, whoops. Silly me (although to be fair, doing all of this on a smartphone isn't easy, heh).